docs/management/api/host-isolation-api.asciidoc

```diff
@@ -7,7 +7,7 @@ Isolates a host running {endpoint-cloud-sec} from the network.
 
 ==== Request URL
 
-`POST <kibana host>:<port>/api/endpoint/isolate`
+`POST <kibana host>:<port>/api/endpoint/action/isolate`
```

Comment:
the old one still exists, right? And it just forwards to the new one while it is "deprecated" but still around.

I think we need to explain that in the docs.

Reply:
good point, will update.